config/initializers/csp.rb

```diff
@@ -7,7 +7,7 @@
     default_src: %w('self'),
     script_src: ["'self'", "'unsafe-eval'", gon_gem_sha, popover_sha],
     font_src: %w('self' fonts.gstatic.com),
-    connect_src: %w('self'),
+    connect_src: %w(*),
```

Comment:
hm. I THINK we want to change this to `'self'` and whatever the env var for our future redis server is but not sure. I'll dig in on this.